utils/utils.go

```diff
@@ -17,9 +17,18 @@ import (
 	"github.com/pborman/uuid"
 )
 
-const testXattr = "trusted.glusterfs.test"
-const volumeIDXattr = "trusted.glusterfs.volume-id"
-const gfidXattr = "trusted.gfid"
+const (
+	testXattr     = "trusted.glusterfs.test"
+	volumeIDXattr = "trusted.glusterfs.volume-id"
+	gfidXattr     = "trusted.gfid"
+)
+
+var (
+	PathMax     = unix.PathMax
```

Comment:
If they are in the same package, all variables are accessible.  Think of the same package as being in the same file.  Only export if you want to make the variables accessible from other packages.